### DISCUSSION TITLE
BIDS format

### DISCUSSION THREAD

**Discussion by hw-ju** [Project-MONAI/tutorials#1396] [Q&A] [answered]
Hi!

Does MONAI support data I/O for [BIDS](bids.neuroimaging.io) format?

**Comment by KumoLiu** [comment]
Hi @hw-ju, to the best of my knowledge, we are not supporting it so far.
Hi @wyli, it would be great if you could double confirm it, thanks in advance!

**Reply by wyli** [reply, marked as answer]
that's not implemented yet, I also suspect it'll be highly task-dependent e.g. https://github.com/KCL-BMEIS/VS_Seg/blob/05065cbd97ebf5366695db5a1c3e8745f2295fb0/preprocessing/createBIDSdataset/data_conversion_BIDS.py

the design of `monai.data.folder_layout` partly considers the use case but it's not a complete solution   https://github.com/Project-MONAI/MONAI/blob/dev/monai/data/folder_layout.py

**Reply by hw-ju** [reply]
Thanks @KumoLiu @wyli for your quick response! 
Will it be supported by MONAI in the near future? Is it on MONAI's roadmap?

**Reply by wyli** [reply]
not at the moment, please feel free to create more detailed feature requests or proposals to dev discussions

**Reply by hw-ju** [reply]
@wyli Gotcha! Thanks :)!

**Reply by hw-ju** [reply]
@wyli Is there a form or structure to use for creating such a feature request?

**Reply by wyli** [reply]
sure, if it's a concrete feature request, please use https://github.com/Project-MONAI/MONAI/blob/dev/.github/ISSUE_TEMPLATE/feature_request.md. if it may require some major developments maybe we can start a discussion to collect the relevant details, like these:
- https://github.com/Project-MONAI/MONAI/discussions/5145
- https://github.com/Project-MONAI/MONAI/discussions/4617

**Reply by hw-ju** [reply]
@wyli Thanks a lot for your help!!
